instrumentation/rails/test/test_helpers/configs/rails6.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module Rails6
+  class Application < Rails::Application
+    config.eager_load = false # Ensure we don't see this Rails warning when testing
+    config.logger = Logger.new('/dev/null') # Prevent tests from creating log/*.log
+    config.hosts << 'example.org'
+    config.secret_key_base = 'secret_key_base'
+  end
+end
+
+def build_app(use_exceptions_controller: false)
+  require 'action_controller/railtie'
+  require 'test_helpers/middlewares'
+
+  new_app = ::Rails6::Application.new
+  new_app.config.secret_key_base = 'secret_key_base'
+
+  # Ensure we don't see this Rails warning when testing
+  new_app.config.eager_load = false
+
+  # Prevent tests from creating log/*.log
+  new_app.config.logger = Logger.new('/dev/null')
+
+  # Required in Rails 6
+  new_app.config.hosts << 'example.org'
+
+  # Creates a lot of deprecation warnins on subsequent app initializations if not explicitly set.
+  new_app.config.action_view.finalize_compiled_template_methods = ActionView::Railtie::NULL_OPTION
```

Comment:
Is this and the `config.hosts` above the only things different between Rails 5 and 6? If so, could you do the version check here and not have the versioned files?

Reply:
Yeah, I've moved some stuff around based on this comment.  I think it's an improvement 🤞